NFproject/NFproject/urls.py

```diff
@@ -21,6 +21,7 @@
 
 urlpatterns = [
     path('', views.home, name='home'),
+    path('all_articles/', views.all_articles, name='all_articles'),
```

Comment:
Make these URLs more conventional. Here's the expected url list (in order):

* `/` goes to the homepage
* `/articles/` goes to a list of all articles
* `/articles/<id>` goes to the page for a specific article
* `/admin` goes to the admin